contracts/.github/workflows/test.yml

```diff
@@ -1,34 +0,0 @@
-name: test
```

Comment:
This file is not used probably created by foundry when initializing the project.